reqs/management/commands/import_reqs.py

```diff
@@ -79,17 +95,18 @@ def priority_split(text, *splitters):
 
 class KeywordProcessor:
     """Creates or retrieves Keyword models"""
-    def __init__(self):
+    def __init__(self, fields):
         self.cache = {}
+        self.fields = fields
 
-    @staticmethod
-    def keywords(row):
+    def keywords(self, row):
         to_return = []
-        for field, value in row.items():
-            if field == 'Other (Keywords)':
+        for field in self.fields:
+            value = row.get(field, None)
```

Comment:
Heads up that with `dict`s you don't need to `d.get(field, None)` -- you can just `d.get(field)` (it'll default to `None` if not present).